### DISCUSSION TITLE
Will the eventual Android/iOS/GTK4 apps support using a different server?

### DISCUSSION THREAD

**Discussion by alexispurslane** [stoatchat/discussions#526] [Feature Suggestions]
I just recently set up a separate server for just my friends and I (for several reasons) and one of the biggest complaints I'm getting from them is the need to use web apps. Is there a plan to allow this? Thank you!

**Comment by Rexogamer** [comment]
I can't comment on the official iOS/Android apps or Mutiny, but RVMob (Android-only at the moment) has basic support for using other instances (the switching experience is a but clunky but it should work well enough)

**Reply by alexispurslane** [reply]
Oh nice, that's good to know!
